nex/node_linux_amd64.go

```diff
@@ -2,35 +2,65 @@ package main
 
 import (
 	"context"
+	"log/slog"
+	"os"
 
 	"github.com/choria-io/fisk"
-	"github.com/sirupsen/logrus"
-	nexnode "github.com/synadia-io/nex/internal/node"
+	// nexnode "github.com/synadia-io/nex/internal/node"
 )
 
 func init() {
-	node_up := nodes.Command("up", "Starts a NEX node")
+	node_up = nodes.Command("up", "Starts a NEX node")
+	node_preflight = nodes.Command("preflight", "Checks system for node requirements and installs missing")
 	node_up.Flag("config", "configuration file for the node").Default("./config.json").StringVar(&NodeOpts.ConfigFilepath)
-	node_preflight := nodes.Command("preflight", "Checks system for node requirements and installs missing")
 	node_preflight.Flag("force", "installs missing dependencies without prompt").Default("false").BoolVar(&NodeOpts.ForceDepInstall)
 	node_preflight.Flag("config", "configuration file for the node").Default("./config.json").StringVar(&NodeOpts.ConfigFilepath)
-	node_up.Action(RunNodeUp)
-	node_preflight.Action(RunNodePreflight)
 
+	ctx := context.Background()
+	opts := slog.HandlerOptions{}
+
+	switch Opts.LogLevel {
+	case "debug":
```

Comment:
May be useful to support `trace` level here also.

Reply:
thats not a slog default, BUT you can add custom cases.  Ill do that in a different PR

Reply:
I openned an issue so i didnt forget.